src/student/student.controller.ts

```diff
@@ -120,6 +120,20 @@ export class StudentController {
     return this.studentService.hireStudent(studentId);
   }
 
+
+
+  @Get("/scheduleStudents")
```

Comment:
Why we have 2 endpoints 
- @Get("/scheduleStudents"
-  @Get('/awaiting-students')
?

Reply:
/scheduleStudents => Returns  schedule students assigned to HR,
awaiting-students' => Returns all schedule students